lib/signaling/v2/iceconnectionmonitor.js

```diff
@@ -94,6 +118,16 @@ class IceConnectionMonitor {
       this._lastActivity = null;
     }
   }
+
+  /**
+   * Stop monitoring the ICE connection state.
+   * and clears _wasInactive state
+   * @returns {void}
+   */
+  clear() {
```

Comment:
@makarandp0 Can we clear the inactive flag in `stop()`. Why do we need a separate method?

Reply:
not applicable anymore.